src/routes/+layout.svelte

```diff
@@ -10,10 +10,10 @@
     $: if (browser) document.documentElement.classList.toggle('dark', $theme);
 </script>
 
-<header class="sticky top-0 lg:top-12">
+<header class="sticky top-0 z-50 lg:fixed lg:inset-x-0 lg:top-10">
```

Comment:
I'm still on the fence about this because `sticky` already decays to the `fixed` positioning once we start scrolling. With that said, I personally don't mind the navbar affecting the document flow because navbars are typically space-occupying components than floating widgets (e.g., a chat bot conversation UI).

Reply:
Honestly, not sure about this change either. As said on the PR, I changed it to `fixed` positioning mainly because the large screen design had the navbar overlapping with the Hero Component and I was trying to make them look more similar to each other. So, I suppose it really depends on whether we want to follow this aspect of the original design or deviate from it. Personally, I think either way is fine, given this change is relatively inconsequential to begin with.

Reply:
I think we can deviate from it in this instance. The overlap feels a little too cramped at the top. 👍

Also, while we're fixing the navbar, could you also add a [`shadow`](https://tailwindcss.com/docs/drop-shadow) around it just so the component is more readable? Feel free to choose whichever `sm`, `md`, `lg`, etc. variant that looks best but most subtle at the same time.